pkg/scheduler/partition_test.go

```diff
@@ -2004,7 +2005,7 @@ func TestPreemption(t *testing.T) {
 	assertPreemptedResource(t, appSummary, -1, 5000)
 
 	appSummary = app2.GetApplicationSummary("default")
-	assertPreemptedResource(t, appSummary, -1, 0)
+	assert.Assert(t, appSummary.PreemptedResource.TrackedResourceMap["UNKNOWN"] == nil)
```

Comment:
Note: since it's no longer `map[string]map[string]int64` but `map[string]*Resource`, referencing a non-existing key results in a panic (this does not happen in the former case, even if `map[string]int64` for a given key was not created with `make()`).

Reply:
Got it!